bokehjs/src/lib/core/properties.ts

```diff
@@ -439,13 +439,17 @@ export abstract class BaseCoordinateSpec<T> extends DataSpec<T> {
 
 export abstract class CoordinateSpec extends BaseCoordinateSpec<number | Factor> {}
 export abstract class CoordinateSeqSpec extends BaseCoordinateSpec<number[] | Factor[]> {}
+export abstract class CoordinateSeqSeqSeqSpec extends BaseCoordinateSpec<number[][][] | Factor[][][]> {}
 
 export class XCoordinateSpec extends CoordinateSpec { readonly dimension = "x" }
 export class YCoordinateSpec extends CoordinateSpec { readonly dimension = "y" }
 
 export class XCoordinateSeqSpec extends CoordinateSeqSpec { readonly dimension = "x" }
 export class YCoordinateSeqSpec extends CoordinateSeqSpec { readonly dimension = "y" }
 
+export class XCoordinateSeqSeqSeqSpec extends CoordinateSeqSeqSeqSpec { readonly dimension = "x" }
+export class YCoordinateSeqSeqSeqSpec extends CoordinateSeqSeqSeqSpec { readonly dimension = "y" }
```

Comment:
Just tossing out as an idea: what do you think of something like `XCoordinateSeqSpec`, `XCoordinateSeq2Spec`, `XCoordinateSeq3Spec`? I actually find `SeqSeqSeqSpec` harded to quickly visually parse

Reply:
I will add this to the overall "naming things" issue.